hystrix/pom.xml

```diff
@@ -24,16 +24,17 @@
         <artifactId>lightblue-mongo-pom</artifactId>
         <version>1.5.0-SNAPSHOT</version>
     </parent>
-    <groupId>com.redhat.lightblue.mongo</groupId>
     <artifactId>lightblue-mongo-hystrix</artifactId>
     <packaging>jar</packaging>
-    <version>1.5.0-SNAPSHOT</version>
```

Comment:
This assumes versions are consistent within a repo.  If that assumption is made then #117 shouldn't have been necessary.  I would like this to be added back in for two reasons: 1) it's explicit, 2) we know it works with the release scripts.  If we decide that all versions will be identical in all modules of a repo then we could remove this assuming the release script works, but I don't think we've explicitly had that discussion yet.